sksparse/cholmod.pyx

```diff
@@ -145,6 +147,37 @@ cdef class Factor
 class CholmodError(Exception):
     pass
 
+class CholmodNotPositiveDefiniteError(CholmodError):
+    def __init__(self, message, column=None, factor=None):
+        super().__init__(message)
+        self.column = column
+        self.factor = factor
+
+class CholmodNotInstalledError(CholmodError):
+    def __init__(self, message=""):
+        message = "method not installed: {}".format(message)
```

Comment:
You are right. I looked in the Cholmod source code. The errors contain already an error message, hence I used them and removed my own error messages.

Yes, I decoded the messages into str.